snuba/clusters/storage_sets.py

```diff
@@ -0,0 +1,33 @@
+from typing import Mapping
+
+from enum import Enum
+from snuba.datasets.storages import StorageKey
+
+
+class StorageSetKey(Enum):
+    EVENTS = "events"
+    OUTCOMES = "outcomes"
+    QUERYLOG = "querylog"
+    SESSIONS = "sessions"
+    TRANSACTIONS = "transactions"
+
+
+STORAGE_SETS: Mapping[StorageKey, StorageSetKey] = {
+    # Events
+    StorageKey.EVENTS: StorageSetKey.EVENTS,
+    StorageKey.ERRORS: StorageSetKey.EVENTS,
+    StorageKey.GROUPEDMESSAGES: StorageSetKey.EVENTS,
+    StorageKey.GROUPASSIGNEES: StorageSetKey.EVENTS,
+    # TODO: Remove once groups are no longer storages
+    StorageKey.GROUPS: StorageSetKey.EVENTS,
+    # Querylog
+    StorageKey.QUERYLOG: StorageSetKey.QUERYLOG,
+    # Outcomes
+    StorageKey.OUTCOMES_RAW: StorageSetKey.OUTCOMES,
+    StorageKey.OUTCOMES_HOURLY: StorageSetKey.OUTCOMES,
+    # Sessions
+    StorageKey.SESSIONS_RAW: StorageSetKey.SESSIONS,
+    StorageKey.SESSIONS_HOURLY: StorageSetKey.SESSIONS,
+    # Transactions
+    StorageKey.TRANSACTIONS: StorageSetKey.TRANSACTIONS,
+}
```

Comment:
I guess I was trying to add the storage set mapping without exposing this anywhere outside the clusters module. The fact that storages are mapped to clusters only through storage sets is a detail that is mostly hidden in here. Kind of subjective but I also find it easier to glance at the mappings in this way rather than open the ~9 files for each storage.

You do need to remember to add it to this list, but it will yell and you and you won't be able to run the API or any command and nearly every test will fail if not.